src/blockdata/block.rs

```diff
@@ -141,9 +141,11 @@ impl BlockHeader {
             return Err(BlockBadTarget);
```

Comment:
Oh, what is this? The target is already retrievable from `self`, so the argument shouldn't be needed?

Reply:
Oh, I was required to _actually_ read the code while refactoring it :(

Reply:
Removed the variant all together.